src/views/FountainView.vue

```diff
@@ -0,0 +1,79 @@
+<template>
```

Comment:
I made changes to BikeCounterView.vue to make it work more universally with table, you can check them here on the commit here https://github.com/HepNuk/GTI525-EQ10/pull/9/commits/637cbe7bfce85976b0f296b5c9f7b094362d4a58

Try to do it for FountainView if you can or lemme know if you have questions